app/Http/Controllers/UserSiteController.php

```diff
@@ -119,4 +135,24 @@ public function sharedUsers(Request $request)
 
         return $this->success($users);
     }
+
+    /**
+     * @param \App\Site $site
+     * @param \Illuminate\Http\Request $request
+     * @return \Illuminate\Http\JsonResponse|\Illuminate\Http\Response
+     */
+    public function toggleReminders(Site $site, Request $request)
+    {
+        $user = $request->user();
+
+        $user_site = UserSite::where('site_id', $site->id)->where('user_id', $user->id)->first();
```

Comment:
This is simpler than returning unauthorized:
```suggestion
        $user_site = UserSite::where('site_id', $site->id)->where('user_id', $user->id)->firstOrFail();
```